app/models/tax_return.rb

```diff
@@ -102,7 +102,8 @@ def standard_deduction
   def outstanding_recovery_rebate_credit
     return nil unless intake.eip1_amount_received && intake.eip2_amount_received
 
-    [expected_recovery_rebate_credit_one - intake.eip1_amount_received, 0].max + [expected_recovery_rebate_credit_two - intake.eip2_amount_received, 0].max
+    benefits = Efile::BenefitsEligibility.new(tax_return: self, dependents: intake.dependents)
```

Comment:
im tempted to move the calculations for outstanding credits and all that jazz into the benefitsEligibility calculation files as well... the reason for this is that when I look at the 1040 and 8812 files, there is a lot of calculation in place happening which feels ... not great. we have to keep that in sync between the pdf version, the xml version, and what we show to clients. when we do it in this file, we need to instantiate a new instance of the benefits eligibility for each method but if we do it in the benefitseligibility class, each place that does any benefits calculations can instantiate one object  and grab all the relevant data from it...

Reply:
yes, that makes sense to me -- i still need to rebase this against the 1040 work that was being done at the same time and want to do that ASAP